go.mod

```diff
@@ -246,6 +246,9 @@ replace (
 	// Required by Cortex https://github.com/cortexproject/cortex/pull/3051.
 	github.com/bradfitz/gomemcache => github.com/themihai/gomemcache v0.0.0-20180902122335-24332e2d58ab
 
+	// TODO(saswatamcode): Remove when https://github.com/efficientgo/tools/pull/14 is merged.
```

Comment:
Merged (:

Reply:
Removed in https://github.com/thanos-io/thanos/pull/5520/commits/4780bf73adc7548693af76af1a058cfd00f6986e 🙂